### PR TITLE
Remove java-repo-tools directory.

### PR DESCRIPTION
**Pull request by tswast**
It is not needed anymore, now that we package the parent pom in Maven
central.

Rather than using submodules, this time I clone the java-repo-tools repo
in the test script. (I probably should have done that for the other
repos, too, but it felt especially important for this quickstart repo).

@lesv PTAL

### CODE REVIEW

**Comment by tswast** [comment]
Oops. I didn't actually delete it. Wait to merge.

**Comment by tswast** [comment]
Okay. Actually removed now. Since the ./travis.sh script clones the repo, the folder reappeared when I tested manually. I added java-repo-tools to the gitignore to prevent that.

File: travis.sh [hunk 1]
```diff
@@ -11,8 +12,25 @@
...
+mvn --batch-mode clean verify | egrep -v "(^\[INFO\] Download|^\[INFO\].*skipping)"
+
+# Run tests using App Engine local devserver.
+test_localhost() {
+  # Pin to a specific commit so that java-repo-tools can be updated
+  # independently of this repo.
+  git clone https://github.com/GoogleCloudPlatform/java-repo-tools.git
+  (
+  cd java-repo-tools
+  git checkout 74755a28f01aedf57992f473eb2d0201b1e04f2a
```
Comment (lesv) [inline travis.sh]: Wow, this looks robust... NOT

Comment (lesv) [inline travis.sh]: I'd rather you used TOT and if it breaks everything, so be it, that's the way G3 mostly works and I think it's relatively sane.

Comment (tswast) [inline travis.sh]: Removed.

**Comment by lesv** [comment]
LGTM - but I no longer see a reference to the repo tools repo.  What magic installs the files?

**Comment by tswast** [comment]
There's still a git clone command in travis.sh.

**Pull request merged**